<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSplitEditor</class>
 <widget class="QWidget" name="NewSplitEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>295</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,3,0,1">
   <item row="0" column="0" colspan="4">
    <layout class="QHBoxLayout" name="hl1" stretch="0,1,0">
     <item>
      <spacer name="hs2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="amountLabel">
       <property name="text">
        <string>Amount</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CreditDebitEdit" name="creditDebitEdit" native="true">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
       <property name="kmm_taborder" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1" rowspan="4">
    <widget class="QPlainTextEdit" name="memoEdit">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="kmm_taborder" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <layout class="QHBoxLayout" name="hl2">
     <item>
      <spacer name="hs3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="enterButton">
       <property name="text">
        <string>Enter</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="kmm_taborder" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="kmm_taborder" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="tagsLabel">
     <property name="text">
      <string>Tags</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="payeeEdit">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAfterCurrent</enum>
     </property>
     <property name="kmm_taborder" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="numberLabel">
     <property name="text">
      <string>Number</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="payToLabel">
     <property name="text">
      <string>Payer/Payee</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KMyMoneyAccountCombo" name="accountCombo">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="kmm_taborder" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="categoryLabel">
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="costCenterLabel">
     <property name="text">
      <string>Costcenter</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="numberEdit">
     <property name="kmm_taborder" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="costCenterCombo">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="kmm_taborder" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="memoLabel">
     <property name="text">
      <string>Memo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" rowspan="2">
    <widget class="KTagContainer" name="tagContainer">
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="kmm_taborder" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMyMoneyAccountCombo</class>
   <extends>QComboBox</extends>
   <header>kmymoneyaccountcombo.h</header>
  </customwidget>
  <customwidget>
   <class>KTagContainer</class>
   <extends>QFrame</extends>
   <header>ktagcontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CreditDebitEdit</class>
   <extends>QWidget</extends>
   <header>creditdebitedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>payeeEdit</tabstop>
  <tabstop>numberEdit</tabstop>
  <tabstop>accountCombo</tabstop>
  <tabstop>costCenterCombo</tabstop>
  <tabstop>memoEdit</tabstop>
  <tabstop>enterButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
